analyses/cell-type-wilms-tumor-14/scripts/summary_results.R

```diff
@@ -89,7 +89,18 @@ arg_df <- tidyr::expand_grid(
 dfs <- purrr::pmap(arg_df,
              \(anchor_assay, level) concat_sample(anchor_assay, level, meta)
 )
+# add rough tumor cell classification, CM = tumor cells, immune/endothelium = normal
 final_table <- dfs %>%
-  purrr::reduce(left_join)
+  purrr::reduce(left_join) %>%
+  mutate(tumor_cell_classification_RNA = case_when(
+    cell_type_assignment_RNA_compartment == "immune" | cell_type_assignment_RNA_compartment == "endothelium" ~ "normal",
+    cell_type_assignment_RNA_compartment == "fetal_nephron" & cell_type_assignment_RNA_celltype == "Cap mesenchyme" ~ "tumor",
+    TRUE ~ "unknown"
```

Comment:
For a bit more clarity (and possibly efficiency), I would use `%in%` here, and separately define the classes that you are using. 

I also updated to using the new `case_when()` `.default` argument.

```suggestion
    cell_type_assignment_RNA_compartment %in% normal_compartments ~ "normal",
    cell_type_assignment_RNA_compartment == "fetal_nephron" & cell_type_assignment_RNA_celltype == "Cap mesenchyme" ~ "tumor",
    .default =  "unknown"
```